proto/src/main/proto/Rss.proto

```diff
@@ -77,6 +77,7 @@ message GetLocalShuffleDataRequest {
   int64 offset = 6;
   int32 length = 7;
   int64 timestamp = 8;
+  int32 storageId = 9;
```

Comment:
We should avoid using storageId. It expose too many details.

Reply:
Do you have any idea?

Reply:
Not yet.